src/modules/get_institution/app/get_institution_controler.ts

```diff
@@ -0,0 +1,45 @@
+import { GetInstitutionUsecase } from "./get_institution_usecase";
+
+import { EntityError } from '../../../core/helpers/errors/EntityError';
+import { Conflict, OK, Unauthorized } from '../../../core/helpers/http/http_codes';
+import { ConflictError, InvalidParameter, InvalidRequest, MissingParameter, UserNotAuthenticated } from '../../../core/helpers/errors/ModuleError';
+import { ParameterError, InternalServerError } from '../../../core/helpers/http/http_codes';
+
+
+export class GetInstitutionController {
+    public usecase: GetInstitutionUsecase;
+
+    constructor(usecase: GetInstitutionUsecase) {
+        this.usecase = usecase;
+    }
+
+    public async execute( id: string): Promise<{ [key: string]: any }> {
+        try {
+            
+            
```

Comment:
The same case! Check in the other routes, what controller receive, what is it logic...

If you have some questions or difficulties, send a message to me!